server/models/post-comment-model.js

```diff
@@ -4,6 +4,9 @@ const { convertTimestampToDate } = require("../../db/seeds/utils.js")
 const { selectArticleById } = require("./get-article-by-id-model.js")
 
 exports.insertComment = (id, bodyArg, authorArg) => {
+    if (!bodyArg || !authorArg) {
+        return Promise.reject({status: 400, msg: "Bad Request"})
+    }
     return db.query(`SELECT article_id FROM articles;`)
```

Comment:
Here you are searcking for all article ids every time, you can just select `WHERE article_id = id` and then if you get one result back you are golden